Sources/TuistKit/ProjectGenerator/ProjectGenerator.swift

```diff
@@ -232,7 +232,12 @@ class ProjectGenerator: ProjectGenerating {
             .workspace
             .merging(projects: updatedGraph.projects.map { $0.path })
 
-        return (updatedWorkspace, project, updatedGraph, modelMapperSideEffects + graphMapperSideEffects)
+        return (
```

Comment:
With 4 returned, I think it's time to group them together in a struct? That way we can access those elements using a more descriptive name (e.g. `returnedType.project`) as opposed to `returnedType.1`.